### PR TITLE
[13.0][MIG] document_page_procedure: Migration to 13.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT20476

### CODE REVIEW

**Comment by ernestotejeda** [comment]
I reduced the commit history a bit

**Comment by ernestotejeda** [comment]
I have removed the **document_page_work_instruction** dependency because this module does not use any features or alters any resources from **document_page_work_instruction**. I don't know if there is any other reason I am missing that I need to maintain that dependence.
ping @pedrobaeza .

**Review by Tardo — APPROVED** [review]
:+1: LGTM

** Update oca_dependencies when ready

**Comment by ernestotejeda** [comment]
oca_dependencies.txt file updated

File: document_page_procedure/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+To be able configure your template of procedures, you need to install
+mgmtsystem_manual:
+
+* go to Management Systems > Configuration > Categories
```
Comment (pedrobaeza) [inline document_page_procedure/readme/CONFIGURE.rst:4]: You can put this with `#. ` numbered list

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Let's go with the merge any way.

/ocabot merge

**Pull request merged**